internal/cli/atlas/customdns/aws/describe_test.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
 	"github.com/mongodb/mongodb-atlas-cli/internal/mocks"
 	"github.com/mongodb/mongodb-atlas-cli/internal/test"
-	atlas "go.mongodb.org/atlas/mongodbatlas"
+	atlas "go.mongodb.org/atlas/mongodbatlasv2"
```

Comment:
super nit
IMHO good pattern is to use atlasv2 or v2 to not confuse us about what is migtated. Same name across the board.
@swattyT @matt-condon

Reply:
[nit] based on https://github.com/mongodb/mongodb-atlas-cli/blob/4e11a40faf0d02a90228f84a303809f2de7a361f/internal/store/access_logs.go it seems like in some places both `mongodbatlas` and `mongodbatlasv2` will exist in the same file. should we stick to`atlasv2` here? ideally could we agree on same naming for all PRs and files?

Reply:
Done, renamed all instances of the v2 client to `atlasv2`